docs/customize/Import-Styles.md

```diff
@@ -23,7 +23,7 @@ customize it.
 
 The core of Nominatim's flex import configuration is the `flex-base` module.
 It defines the table layout used by Nominatim and provides standard
-implementations for the import callbacks that help with customizing
+implementations for the import callbacks that help with customising
```

Comment:
I was trying to settle on American spelling on the z vs. s front. In line with variable naming in the code.

Reply:
Ah, ok, I saw "recognise" and "initalise" later in the document and thought it was vice-versa.